### ISSUE TITLE
export_data new error

### ISSUE THREAD

**Issue by alexpunct** [timelyart/Kairos#34]
I started seeing this error with the first symbols in the watchlist

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kairos/Kairos/tv/tv.py", line 1880, in retry_process_symbol
    input_symbol.send_keys(Keys.ENTER)
  File "/usr/local/lib/python3.6/dist-packages/selenium-3.141.0-py3.6.egg/selenium/webdriver/remote/webelement.py", line 479, in send_keys
    'value': keys_to_typing(value)})
  File "/usr/local/lib/python3.6/dist-packages/selenium-3.141.0-py3.6.egg/selenium/webdriver/remote/webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "/usr/local/lib/python3.6/dist-packages/selenium-3.141.0-py3.6.egg/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.6/dist-packages/selenium-3.141.0-py3.6.egg/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
  (Session info: headless chrome=91.0.4472.101)

2021-11-10 06:41:42.776 INFO tv - process_symbol: BINANCE:1INCHDOWNUSDT
2021-11-10 06:41:43.278 ERROR tv - process_symbol: 'export_data'
Traceback (most recent call last):
  File "/home/kairos/Kairos/tv/tv.py", line 1857, in process_symbol
    export_enabled = 'enabled' in chart['export_data'] and chart['export_data']['enabled'] and \
KeyError: 'export_data'
```

**Comment by timelyart** [comment]
Can you give a bit more context? Like what were you trying to do? Did Kairos make screenshot? Were you running in background mode?

**Comment by timelyart** [comment]
It seems like some sort of dialog was getting in the way of things. Do you know which dialog? A screenshot would be extremely helpful.

**Comment by alexpunct** [comment]
Yes, it's running in background mode on linux so I can't give you a screenshot. I haven't changed anything, it stopped working since today, it looks like it does it as soon as it starts browsing through the watchlist:

```2021-11-10 13:25:57.975 INFO tv - run: Running on a linux operating system
2021-11-10 13:25:59.593 INFO tv - check_driver: browser version: 95.0.4638.69
2021-11-10 13:25:59.593 INFO tv - check_driver: driver version: 95.0.4638.69
2021-11-10 13:26:13.203 INFO tv - login: logged in successfully at tradingview.com as alexpunct
2021-11-10 13:26:13.412 INFO tv - open_chart: opening chart https://www.tradingview.com/chart/lbeSBG8s/
2021-11-10 13:26:13.413 INFO tv - open_chart: WAIT_TIME_IMPLICIT = 0.0
2021-11-10 13:26:13.413 INFO tv - open_chart: PAGE_LOAD_TIMEOUT = 30.0
2021-11-10 13:26:13.413 INFO tv - open_chart: CHECK_IF_EXISTS_TIMEOUT = 10.0
2021-11-10 13:26:13.413 INFO tv - open_chart: DELAY_BREAK_MINI = 0.2
2021-11-10 13:26:13.413 INFO tv - open_chart: DELAY_BREAK = 0.5
2021-11-10 13:26:13.414 INFO tv - open_chart: DELAY_SUBMIT_ALERT = 2.5
2021-11-10 13:26:13.414 INFO tv - open_chart: DELAY_CHANGE_SYMBOL = 0.2
2021-11-10 13:26:13.414 INFO tv - open_chart: DELAY_CLEAR_INACTIVE_ALERTS = 0.0
2021-11-10 13:26:13.415 INFO tv - open_chart: DELAY_KEYSTROKE = 0.05
2021-11-10 13:26:13.415 INFO tv - open_chart: DELAY_READ_INDICATOR_VALUE = 0.2
2021-11-10 13:26:13.415 INFO tv - open_chart: READ_FROM_DATA_WINDOW = True
2021-11-10 13:26:13.416 INFO tv - open_chart: WAIT_UNTIL_CHART_IS_LOADED = True
2021-11-10 13:26:13.416 INFO tv - open_chart: READ_ALL_VALUES_AT_ONCE = True
2021-11-10 13:26:13.416 INFO tv - open_chart: CHANGE_SYMBOL_WITH_SPACE = False
2021-11-10 13:26:13.416 INFO tv - open_chart: VERIFY_MARKET_LISTING = True

2021-11-10 13:26:29.063 INFO tv - open_chart: Binance Futures USDT: 78 markets found
2021-11-10 13:26:29.516 INFO tv - set_timeframe: Setting timeframe to 1 hour
2021-11-10 13:26:31.620 INFO tv - open_chart: opening watchlist Binance Futures USDT
2021-11-10 13:26:43.797 INFO tv - process_symbol: BINANCE:AAVEUSDT
2021-11-10 13:26:45.772 ERROR tv - process_symbol: 'export_data'
Traceback (most recent call last):
  File "/home/kairos/Kairos/tv/tv.py", line 1857, in process_symbol
    export_enabled = 'enabled' in chart['export_data'] and chart['export_data']['enabled'] and \
KeyError: 'export_data'
2021-11-10 13:26:45.774 INFO tv - retry_process_symbol: trying again (1)
2021-11-10 13:26:56.829 ERROR tv - retry_process_symbol: Message: element not interactable
  (Session info: headless chrome=95.0.4638.69)
Traceback (most recent call last):
  File "/home/kairos/Kairos/tv/tv.py", line 1857, in process_symbol
    export_enabled = 'enabled' in chart['export_data'] and chart['export_data']['enabled'] and \
KeyError: 'export_data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kairos/Kairos/tv/tv.py", line 1880, in retry_process_symbol
    input_symbol.send_keys(Keys.ENTER)
  File "/usr/local/lib/python3.6/dist-packages/selenium-3.141.0-py3.6.egg/selenium/webdriver/remote/webelement.py", line 479, in send_keys
    'value': keys_to_typing(value)})
  File "/usr/local/lib/python3.6/dist-packages/selenium-3.141.0-py3.6.egg/selenium/webdriver/remote/webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "/usr/local/lib/python3.6/dist-packages/selenium-3.141.0-py3.6.egg/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.6/dist-packages/selenium-3.141.0-py3.6.egg/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable
  (Session info: headless chrome=95.0.4638.69)```

**Comment by timelyart** [comment]
Ah - I see. I done goofed up. 

Fixed in [release v2.58](https://github.com/timelyart/Kairos/releases/tag/v2.58).
